ffi/athcon/bindings/rust/athcon-vm/src/lib.rs

```diff
@@ -253,6 +261,17 @@ impl<'a> ExecutionContext<'a> {
     } else {
       std::ptr::null()
     };
+    let method = message.method();
+    let method_name_size = if let Some(method) = method {
+      method.len()
+    } else {
+      0
+    };
+    let method_name = if let Some(method) = method {
+      method.as_ptr()
+    } else {
+      std::ptr::null()
+    };
```

Comment:
NIT (I know it's the same as above - perhaps update it too?)
```suggestion
    let (method_name, method_name_size) = if let Some(method) = message.method() {
      (method.as_ptr(), method.len())
    } else {
      (std::ptr::null(), 0)
    };
```